client/src/App.vue

```diff
@@ -84,9 +71,45 @@
         snackbar: false,
         snackbarMessage: '',
         snackbarStyle: '',
-        activeMenuItem: ''
+        activeMenuItem: '',
+        player: null,
+        playerState: {
+          paused: true,
+          repeat: false,
+          shuffle: false,
+          position: 0,
+          track: 'Track Name',
+          artist: 'Artist Name',
+          images: [{}],
+          elapsed: '00:00',
+          duration: '00:00',
+          durationMs: 0,
+          volume: 50
+        }
       }
     },
+    beforeCreate () {
+      WebPlaybackService.getPlayer().then(player => {
```

Comment:
We should consider keeping this logic in the WebPlaybackService. Maybe a method like `subscribeComponentToPlayerChanges(component)` that we could call with `WebPlaybackService.subscribeComponentToPlayerChanges(this)`. Then we keep all the logic in the service and we can keep `WebPlaybackService.getPlayer()` private.